pkg/porter/archive_test.go

```diff
@@ -51,3 +52,18 @@ func TestArchive_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestArchive_ArchiveDirectory(t *testing.T) {
+	p := NewTestPorter(t)
+	defer p.Close()
+	ex := exporter{
+		fs: p.FileSystem,
+	}
+
+	dir, err := ex.createArchiveFolder("examples/test-bundle-0.2.0")
+	require.NoError(t, err)
+
+	info, err := ex.fs.Stat(dir)
+	require.NoError(t, err)
+	require.Equal(t, pkg.FileModeDirectory.String(), info.Mode().Perm().String(), dir)
```

Comment:
We also have a test helper that gives you a better error message whe comparing file modes 

```
import "get.porter.sh/porter/tests"

... 

tests.AssertDirectoryPermissionsEqual(...)
```

Reply:
You don't have to change the code to use it, I just wanted you to be aware of the helpers since they are spread out. We really should consolidate them eventually! 😅

Reply:
Oh, I did not notice we have this nice helper function. Thank your for mentioning it. I just changed to use this helper funcion cuz it's so nice